Require/index.js

```diff
@@ -209,6 +209,6 @@ describe("TNS require", function () {
 
     it("should handle JSON file errors", function () {
         require('./RequireJsonCorruptFile');
-        expect(TNSGetOutput()).toBe("File: 'file:///app/shared/Require/RequireJsonCorruptFile/test.json'. JSON Parse error: Unterminated string");
+        expect(TNSGetOutput()).toBe("There was a corrupt or invalid package.json.");
```

Comment:
tested with: `.toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)` and it works for android when the TnsLog is reverted to the way it was before -> `TNSLog(e.message);`